packages/api-contract/src/method.ts

```diff
@@ -2,12 +2,12 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { CodecArg, Constructor } from '@polkadot/types/types';
-import { ContractABIFn, ContractABIFn$Arg, ContractABIMethod, ContractABIMethodBase, ContractABITypes } from './types';
-
 import { Compact, createClass } from '@polkadot/types';
+import { CodecArg, Constructor } from '@polkadot/types/types';
 import { assert, isString, isUndefined, stringCamelCase } from '@polkadot/util';
 
+import { ContractABIFn, ContractABIFn$Arg, ContractABIMethod, ContractABIMethodBase, ContractABITypes } from './types';
```

Comment:
This is actually wrong, which is why we have not enabled any rules.

- type imports first
- external imports, i.e. bn.js
- @polkadot imports
- local imports